youtube_dl/extractor/telewebion.py

```diff
@@ -25,31 +25,26 @@ class TelewebionIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        secure_token = self._download_webpage(
-            'http://m.s2.telewebion.com/op/op?action=getSecurityToken', video_id)
-        episode_details = self._download_json(
-            'http://m.s2.telewebion.com/op/op', video_id,
-            query={'action': 'getEpisodeDetails', 'episode_id': video_id})
+        episode_details = self._download_json('https://gateway.telewebion.ir/kandoo/episode/getEpisodeDetail/?EpisodeId={}'.format(video_id), video_id)
+        episode_details = episode_details["body"]["queryEpisode"][0]
 
-        m3u8_url = 'http://m.s1.telewebion.com/smil/%s.m3u8?filepath=%s&m3u8=1&secure_token=%s' % (
-            video_id, episode_details['file_path'], secure_token)
+        channel_id = episode_details["channel"]["descriptor"]
+        episode_image_id = episode_details.get("image")
+        episode_image = "https://static.telewebion.com/episodeImages/{}/default".format(episode_image_id)
+
+        m3u8_url = 'https://cdna.telewebion.com/{channel_id}/episode/{video_id}/playlist.m3u8'.format(channel_id=channel_id, video_id=video_id)
         formats = self._extract_m3u8_formats(
             m3u8_url, video_id, ext='mp4', m3u8_id='hls')
 
-        picture_paths = [
-            episode_details.get('picture_path'),
-            episode_details.get('large_picture_path'),
-        ]
-
         thumbnails = [{
-            'url': picture_path,
-            'preference': idx,
-        } for idx, picture_path in enumerate(picture_paths) if picture_path is not None]
+            'url': episode_image,
+        }]
 
         return {
             'id': video_id,
-            'title': episode_details['title'],
+            'title': episode_details.get('title'),
             'formats': formats,
             'thumbnails': thumbnails,
```

Comment:
Scrap the `thumbnails` stuff and replace the returned value with a single `thumbnail`
```py
            'thumbnail': url_or_none(episode_image),
```
(we expect it will pass the `url_or_none` -- import from `..utils` -- test since it was constructed earlier, but that might change)